torch/autograd/__init__.py

```diff
@@ -147,13 +150,13 @@ def backward(
         tuple(inputs) if inputs is not None else tuple()
 
     grad_tensors_ = _tensor_or_tensors_to_tuple(grad_tensors, len(tensors))
-    grad_tensors_ = _make_grads(tensors, grad_tensors_)
+    grad_tensors_ = _make_grads(tensors, grad_tensors_, batched_grads=False)
     if retain_graph is None:
         retain_graph = create_graph
 
     Variable._execution_engine.run_backward(
```

Comment:
Still think that the best would be to just collapse these three lines into a single one for fixing https://github.com/pytorch/pytorch/issues/41659 (even if this needs suppressing the syntax linter for this particualr case)

Reply:
Hmm, I'm okay with this